src/Task/Task.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { AiFillEdit, AiOutlineCheck, AiOutlineClose } from "react-icons/ai";
+import './Task.css';
+
+const Task = (props) => {
+
+  // Flag for highlight the card
+  const [classFlag, setClassState] = useState(true);
+
+  const selectItem = () => setClassState(!classFlag);
+
+  // Stage for type of button and adding input fields
+  let [buttonFlag, setTaskState] = useState(true);
+
+  const changeText = () => {
+    if (classFlag === false) setClassState(!classFlag)
```

Comment:
Why so complicated if you'll have `classFlag = true` after that piece of code anyway? You can just forcibly set it to `true`.